mungegithub/mungers/submit-queue.go

```diff
@@ -72,7 +72,8 @@ const (
 var (
 	_ = fmt.Print
 	// This MUST cause a RETEST of everything in the sq.RequiredRetestContexts
-	retestBody = fmt.Sprintf("@%s test this [submit-queue is verifying that this PR is safe to merge]", jenkinsBotName)
+	newRetestBody = "/test all [submit-queue is verifying that this PR is safe to merge]"
+	oldRetestBody = fmt.Sprintf("@%s test this [submit-queue is verifying that this PR is safe to merge]", jenkinsBotName)
```

Comment:
yes, having a TODO to remove `oldRetestBody` when the time comes is a good suggestion. :)